propagator/xray/lib/opentelemetry/propagator/xray.rb

```diff
@@ -46,3 +46,4 @@ def text_map_propagator
 end
 
 require_relative './xray/version'
+require_relative './id_generator'
```

Comment:
```suggestion
require_relative './xray/id_generator'
```
and rename the file accordingly